sdks/java/io/debezium/build.gradle

```diff
@@ -54,8 +54,14 @@ dependencies {
     permitUnusedDeclared "org.apache.kafka:connect-json:2.5.0" // BEAM-11761
 
     // Debezium dependencies
-    compile group: 'io.debezium', name: 'debezium-core', version: '1.3.1.Final'
-    testCompile group: 'io.debezium', name: 'debezium-connector-mysql', version: '1.3.1.Final'
+    def debezium_version = '1.3.1.Final'
+    compile group: 'io.debezium', name: 'debezium-core', version: debezium_version
+    compile group: 'io.debezium', name: 'debezium-connector-mysql', version: debezium_version
+    compile group: 'io.debezium', name: 'debezium-connector-postgres', version: debezium_version
+    compile group: 'io.debezium', name: 'debezium-connector-sqlserver', version: debezium_version
+    compile group: 'io.debezium', name: 'debezium-connector-oracle', version: debezium_version
+    compile group: 'io.debezium', name: 'debezium-connector-db2', version: debezium_version
```

Comment:
It would be good to make these dependencies optional, because otherwise deploying these pipelines may require a lot of artifacts, and could lead to dependency issues. Do you know if it's possible to specify extra dependencies optionally with x lang transforms?

cc: @chamikaramj

Reply:
I don't know how to specify extra dependencies with x lang transforms, this is the only part of comments missing to upload a new commit.